src/test/java/org/apache/onami/persist/test/transaction/testframework/TransactionalWorker.java

```diff
@@ -109,7 +109,7 @@ public void doTasks() {
   /**
    * Check all stored entities if they actually have been persisted in the DB.
    */
-  @Transactional
+  @Transactional(onUnits = {})
```

Comment:
These tests are not tests, but Tasks, they are called from other tests which does randomized testing.